src/app/payment/paymentMethod/getPaymentMethodName.ts

```diff
@@ -56,6 +56,10 @@ export function getTranslatedPaymentMethodName(
             return language.translate('payment.amazon_name_text');
         }
 
+        if (method.id === PaymentMethodId.Bolt) {
+            return language.translate('payment.bolt_name_text');
```

Comment:
https://app.circleci.com/pipelines/github/bigcommerce/checkout-js/3357/workflows/1a47ca2e-d97f-4aac-bf20-87d6256250dc/jobs/8428 :( This method started off with just a few `if` but now it has more than 10. Maybe it's a sign that we need to revisit the implementation. Perhaps maybe have a dictionary like

```tsx
const translations = {
    [PaymentMethodId.Bolt]: language.translate('payment.bolt_name_text'),
    [PaymentMethodId.Amazon]: language.translate('payment.amazon_name_text'),
    // ...
}
```
And then you can read the translation by `translations[method.id]`. That way, we can remove all the `if` and reduce the cyclomatic complexity.

Reply:
I think that it is a good idea

Reply:
Changed